### ISSUE TITLE
Set `imageShouldStartVisible` and `showInitialsUntilImageLoads` to true by default in Persona

### ISSUE THREAD

**Issue by hjonasson** [microsoft/fluentui#15945]
#### Describe the feature that you would like added

The `Persona` component has a lot of flickering in various components for us (see below). We have solved this by setting `imageShouldStartVisible` and `showInitialsUntilImageLoads` to true but these seem preferable as default behaviour.

![image](https://user-images.githubusercontent.com/7723013/99226625-5bbee600-27ea-11eb-953f-a147b5d9bc9e.png)

#### What component or utility would this be added to

The Persona component, notably [here](https://github.com/microsoft/fluentui/blob/c6ae2b350037deb4e0bea07cdbdfe2e93455f4b1/packages/react-internal/src/components/Persona/Persona.base.tsx#L24) (or in some other way if you find a better idea).

#### Have you discussed this feature with our team, and if so, who

I am using this ticket to bring it up

#### Additional context/screenshots

This would change this flickering, internally called the Christmas tree to a much subtler transition. I have a gif but github keeps changing it to a png.

**Comment by paulgildea** [comment]
@hjonasson Thanks for filing this issue.

Just to understand your request a little bit more, can you send us a codepen of the flickering issue that you are seeing today as well as the version numbers of the package you are using? I'd like to understand if this is an issue with the component itself or the use case.

And it sounds like you have the behavior you want, it just that it's not the default behavior correct?

**Comment by paulgildea** [comment]
@JustSlone FYI

**Comment by hjonasson** [comment]
I might actually be the use case. I don't think so but it might be. If it is then I will close the ticket.

I am trying to repro it in codepen but I don't get it to render initials before the image. That transition is the jarring experience. 

You are correct, this is just a feature request, with the flags I mention we get the behaviour we want.


Here is the code I used in CodePen, starting from the [Fabric Starting Template](https://codepen.io/FabricReact/pen/NvBvWx). Creating a new pen with the same code kept erroring.
```TS
let { Persona, Fabric } = window.Fabric;

const Content = () => {
  const onRenderPrimaryText = React.useCallback(() => <>{"Nice Puffin"}</>, [
    "Nice Puffin"
  ]);

  return (
    <Fabric className="foo">
      <Persona
        text="Nice Puffin"
        onClick={() => alert("hello")}
        onRenderPrimaryText={onRenderPrimaryText}
        imageUrl="https://cdn.jpegmini.com/user/images/slider_puffin_before_mobile.jpg"
      ></Persona>
    </Fabric>
  );
};

ReactDOM.render(<Content />, document.getElementById("content"));
```

**Comment by JustSlone** [comment]
Here's an updated pen comparing the cases, and a gif below (on slow 3g simulation)
https://codepen.io/justslone-the-encoder/pen/RwROjVx
![persona](https://user-images.githubusercontent.com/40304143/99492619-c2dfb600-2922-11eb-9896-3c00c01db42e.gif)

@hjonasson thanks for the suggestion, it's a good idea. As far changing Persona's defaults, we're probably not going to be able to change the defaults right now since that would be a breaking change. And we're also working on Avatar which is an updated Persona, so we should consider this for Avatar, but I just checked and good news these are the defaults for Avatar!

Here's what Avatar looks like by default
https://codesandbox.io/s/avatar-example-e3hun
![avatar](https://user-images.githubusercontent.com/40304143/99494186-6e8a0580-2925-11eb-962c-47598abe016d.gif)

Seems like we should close this out, thanks for submitting the issue though I'm glad we made sure Avatar had this behavior by default.
